app/jobs/moab_to_catalog_job.rb

```diff
@@ -11,8 +11,9 @@ class MoabToCatalogJob < ApplicationJob
 
   # @param [MoabStorageRoot] root mount containing the Moab
   # @param [String] druid
-  # @param [String] path
-  def perform(root, druid, path)
+  # @param [String] optional path
+  def perform(root, druid, path=nil)
+    path = "#{root.storage_location}/#{DruidTools::Druid.new(druid).tree.join('/')}" if path.nil?
```

Comment:
question for @jmartin-sul:  do we want to warn before we do this lookup?

Reply:
i don't think that's necessary... to my knowledge, there's no appreciable difference in overhead (we've already got the storage root object, and AFAIK it should have all its fields populated by the query that instantiated it).  and even if we wanted to warn because the lack of the param implies manual remediation (per #1158), i'd argue for a flag specifically indicating that, since i'd agree w/ @mjgiarlo that the param is probably superfluous.